### PR TITLE
DataTable: data info displayed in the status bar

### PR DESCRIPTION
**Pull request by aturanjanin**
##### Description of changes
Input/output data info displayed in the status bar.

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

File: Orange/widgets/data/owtable.py [hunk 1]
```diff
@@ -884,6 +876,10 @@ def commit(self):
             # Selections of individual instances are not implemented
             # for SqlTables
             if isinstance(table, SqlTable):
+                summary = selected_data.approx_len() if selected_data else \
```
Comment (VesnaT) [inline Orange/widgets/data/owtable.py:879]: `selected_data` is set to None at the beginning of the function and never set to anything else.

File: Orange/widgets/data/owtable.py [hunk 2]
```diff
@@ -1028,70 +1027,6 @@ def parts(array, density, col_dist):
         return Summary(n_instances, domain, X_part, Y_part, M_part)
 
 
-def format_summary(summary):
```
Comment (VesnaT) [inline Orange/widgets/data/owtable.py:1031]: Try reusing this function (or rewrite it for your needs) to get more accurate results on lengths.

File: Orange/widgets/data/owtable.py [hunk 3]
```diff
@@ -491,6 +485,7 @@ def sizeHint():
         return QSize(800, 500)
 
     @Inputs.data
+    @check_sql_input
```
Comment (VesnaT) [inline Orange/widgets/data/owtable.py:488]: I think this decorator was left out intentionally. 
The widget handles big sql tables differently then other widget.

File: Orange/widgets/data/owtable.py (hunk 2 above)
Comment (aturanjanin) [reply Orange/widgets/data/owtable.py:1031]: What do you think about reusing this function but within the `format_summary_details` function? That function takes the table as the input and it calculates all the lengths itself. Also, in that way we get consistent results for all widgets.

Comment (VesnaT) [reply Orange/widgets/data/owtable.py:1031]: I only meant it for the "length" part of the function. It returns `approx_len` for SqlTable, unless the length has finished calculating. I'm not sure whether we can spot the difference (probably for big sql tables), but I guess we should keep the behavior.

**Comment by VesnaT** [comment]
Can you pleas rebase to master?

**Comment by VesnaT** [comment]
Can you pleas rebase to master?

**Comment by aturanjanin** [comment]
@VesnaT Rebased now, thanks.

**Pull request merged**